content/docs/application-mobility/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Application Mobility"
+linkTitle: "Application Mobility"
+weight: 9
+Description: >
+  Application Mobility for Dell Container Storage Modules (CSM)
```

Comment:
Shouldn't it be "Dell Container Storage Modules (CSM) for Application Mobility" or just "Application Mobility"?

Reply:
Updated this and others to just the title of the page to remain consistent